Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "cloudbbq2-rs"
```

Comment:
We could actually rename it to cloudbbq. I only named it cloudbbq2 because the first attempt was using a different bt library.

Reply:
Done.